src/clj/reviews_next/config.clj

```diff
@@ -5,3 +5,9 @@
 
 (defstate config
   :start (read-config (io/resource "config.edn")))
+
+(defn connection-uri
```

Comment:
This function can be modified to take (config/connection-uri) and arity can be removed.